src/lib/core/utils/default-dependent-axis-range.ts

```diff
@@ -0,0 +1,98 @@
+import { Variable } from '../types/study';
+import { NumberOrDateRange } from '@veupathdb/components/lib/types/general';
+
+export function defaultDependentAxisRange(
+  variable: Variable | undefined,
+  plotName: string,
```

Comment:
I noticed that `plotName` is unused.
Well actually my editor did, lol...

Reply:
@bobular first of all thank you for your reviews! I just leave plotName props in there as it may be possibly used for other plot, e.g., line plot, density plot, etc.